app/server/src/main/scala/org/bitcoins/server/BitcoinSServerMain.scala

```diff
@@ -256,12 +177,13 @@ object Main extends App with HttpLogger {
   }
 
   //start everything!
-  val run = runMain(args.toVector)
-
-  run.failed.foreach { err =>
-    logger.error(s"Failed to startup server!", err)
-    sys.exit(1)
-  }(scala.concurrent.ExecutionContext.Implicits.global)
+  lazy val run: Unit = {
```

Comment:
Why is this a `lazy val` ? Maybe make this a `def run(): Unit` ?

Reply:
Made it a lazy val so we can't accidentally start the server twice